acceptance-tests/Dockerfile

```diff
@@ -30,6 +30,9 @@ RUN python3 -m pip install \
     'seaborn==0.9.0' \
     git+https://github.com/SwissDataScienceCenter/renku-python.git@${renku_python_ref}
 
+RUN wget --quiet https://dl.min.io/client/mc/release/linux-amd64/mc -O /usr/local/bin/mc && \
```

Comment:
does this need a separate `RUN` or can it be bundled into the one on line 14?

Reply:
Yes, I'll do that.